lib/associations/base.js

```diff
@@ -1,3 +1,21 @@
-var Association = function () {};
+'use strict';
 
-module.exports = Association;
\ No newline at end of file
+var Association = function() {};
+
+// Normalize input - may be array or single obj, instance or primary key - convert it to an array of built objects
+Association.prototype.toInstanceArray = function (objs) {
+  if (!Array.isArray(objs)) {
+    objs = [objs];
+  }
+  return objs.map(function(obj) {
+    if (!(obj instanceof this.target.Instance)) {
+      var tmpInstance = {};
+      tmpInstance[this.target.primaryKeyAttribute] = obj;
+      return this.target.build(tmpInstance, {
```

Comment:
I'm not really sure if this is such a good idea. Many developers won't have a clue what that stands for and you prevent other's from working on sequelize. Every JS developer should know the old syntax but only a few might know about the newest features. So is it really worth it?